third_party/rust/Cargo.lock

```diff
@@ -77,9 +73,57 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
```

Comment:
what are all these new dependencies?

Reply:
`anstyle` is used by `clap_builder` which is in upstream chromium. So is `anstream` but it's optional, so I'm not sure why we're pulling it in. Maybe the thing where cargo locks versions for optional dependencies even if they're not built?

I'll see if we really need the direct dependency on `clap`.

`bitflags` is a dep of `adblock` and is in upstream chromium.
`bech32` was added recently for zcash support.